opendcs-web-client/build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 dependencies {
-    implementation(libs.servlet.api)
+    implementation(libs.jetty.jsp)
```

Comment:
A jetty dependency here is suspicious given this should also run in the context of tomcat

Reply:
I created issue #92 to track this.